docs/src/languages/php/xdebug.md

```diff
@@ -68,31 +80,34 @@ your requests don't always reach the same host.
 ### Install an Xdebug helper
 
 While Xdebug can be triggered from the browser by adding a special query parameter, the preferred way is to use a browser plugin helper.
-One is available for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/xdebug-helper-for-firefox/) and for [Chrome](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc).
+Helpers are available for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/xdebug-helper-for-firefox/)
+and [Chrome](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc).
 Their respective plugin pages document how to trigger them when needed.
 
-## Using PHPStorm
+## 3. Optional: Use PHPStorm
```

Comment:
:hammer_and_wrench: 
```suggestion
## 3. Configure your IDE

Follow the instructions for your IDE, such as those for [PHPStorm](https://www.jetbrains.com/help/phpstorm/configuring-xdebug.html.
```

And then delete the rest of this step and its confusing numbers within numbers. Only keep anything Platform.sh-specific.